app/controllers/stripe-setup/vat-number-company-number/vat-number/get.controller.js

```diff
@@ -8,9 +8,14 @@ const response = require('../../../../utils/response')
 
 module.exports = (req, res) => {
   // initialise pageData
-  let pageData = {
-    errors: lodash.get(req, 'session.stripeSetup.vatNumber.errors'),
-    vatNumber: lodash.get(req, 'session.stripeSetup.vatNumber.vatNumber', '')
+  let pageData = lodash.get(req, 'session.pageData.stripeSetup.vatNumber')
+  if (pageData) {
+    delete req.session.pageData.stripeSetup.vatNumber
+  } else {
+    pageData = {
+      errors: {},
+      vatNumber: ''
```

Comment:
probably not necessary to set anything in pageData in this case? but not a big deal

Reply:
yes, I was wondering the same, but thought that it is good to know the structure of it if you think it is more confusing to have the blank values?